queries.sql

```diff
@@ -16,5 +16,47 @@ SELECT name, FROM animals WHERE name != 'Gabumon';
 
 SELECT name FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+/*-----------------------------------------------------------------*/
+BEGIN;
+UPDATE animals SET species = 'unspecified' WHERE species IS NULL;
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
```

Comment:
- [ ] according to the requirements you should "Inside a transaction update the animals table by setting the species column to unspecified" you don't need to specify the part `WHERE species IS NULL`

Reply:
Sorry that is assertive @mikethreels  i fixed it
![image](https://user-images.githubusercontent.com/105387310/206040128-e50d6104-29c2-4184-ade9-e04d4a39e6cd.png)